plugins/sysinfo_plugin/tests/sysinfo_plugin_test.py

```diff
@@ -0,0 +1,531 @@
+    assert_equal(request_string, get_response(response), {'error': 'Minimal interval value is 5 minutes'},
+                 test_name)
+
+    test_name = "end time less than start time"
+    payload = {
+        "run": {
+            "startTime": datetime_start.strftime(DATETIME_FORMAT),
+            "endTime": (datetime_end - timedelta(seconds=10)).strftime(DATETIME_FORMAT),
+            "interval": 5
+        }
+    }
+    response, request_string = make_request(POST, COMPARE, payload=payload)
+    assert_equal(request_string, get_code(response), 400, test_name)
+    assert_equal(request_string, get_response(response), {'error': 'End time is less than current time'},
+                 test_name)
+
+    datetime_end = datetime_start = get_server_datetime() + timedelta(seconds=5)
+    good_payload = {
+        "run": {
+            "startTime": datetime_start.strftime(DATETIME_FORMAT),
+            "endTime": datetime_end.strftime(DATETIME_FORMAT),
+            "interval": 10
+        }
+    }
+    response, request_string = make_request(POST, COMPARE, payload=good_payload)
+    assert_equal(request_string, get_code(response), 200)
+    assert_equal(request_string, get_response(response), {})
+
+    if get_code(response) == 200:
+        time.sleep(5)
+        check_comparison_report("Periodic")
+
+    test_name = "start scheduler twice"
+    datetime_end = datetime_start = get_server_datetime() + timedelta(seconds=2)
+    datetime_end = datetime_start + timedelta(minutes=10)
+    payload = {
+        "run": {
+            "startTime": datetime_start.strftime(DATETIME_FORMAT),
+            "endTime": datetime_end.strftime(DATETIME_FORMAT),
+            "interval": 5
+        }
+    }
+    make_request(POST, COMPARE, payload=payload)
+    response, request_string = make_request(POST, COMPARE, payload=payload)
+    assert_equal(request_string, get_code(response), 400, test_name)
+    assert_equal(request_string, get_response(response), {'error': 'Periodic comparison is already running'}, test_name)
+
+    response, request_string = make_request(POST, CANCEL)
+    assert_equal(request_string, get_code(response), 200)
+    assert_equal(request_string, get_response(response), {})
+
+    test_name = "cancel nothing"
+    response, request_string = make_request(POST, CANCEL)
+    assert_equal(request_string, get_code(response), 400, test_name)
+    assert_equal(request_string, get_response(response), {'error': 'Periodic comparison is not running'}, test_name)
+
+
+def delete_ndts(ndts_folder):
+    print("Delete NDTs test")
+    delete_request = []
+    for ndt in os.listdir(ndts_folder):
+        delete_request.append({"file_name": "{}".format(ndt)})
+
+    test_name = "not allowed"
+    response, request_string = make_request(GET, DELETE)
+    assert_equal(request_string, get_code(response), 405, test_name)
+    assert_equal(request_string, get_response(response), {'error': 'Method is not allowed'}, test_name)

```

Comment:
constants, anything you are using it more that one time it's better to use it as constant